build.gradle

```diff
@@ -114,7 +115,6 @@ dependencies {
     implementation "com.github.luben:zstd-jni:1.5.5-3"
 
     implementation "org.slf4j:slf4j-api:1.7.36"
-    implementation "com.google.guava:guava:32.0.0-jre"
```

Comment:
Dropping it unless we have a good reason to keep this dep. I see it used on GCS, but we could manage it there instead.